romanisim/l3.py

```diff
@@ -275,7 +275,7 @@ def l3_psf(bandpass, scale=0, chromatic=False, **kw):
     return psf
 
 
-def simulate(shape, wcs, efftimes, filter_name, catalog, nexposures=1,
+def simulate(shape, wcs, efftimes, filter_name, sca, catalog, nexposures=1,
```

Comment:
For L3 we don't want to have an SCA argument.  Elsewhere we assume some default SCA, romanisim.parameters.default_sca.  Let's continue doing that here?  If we really want to specify this at L3, let's make sca have a default of None that then gets set to romanisim.parameters.default_sca.

Reply:
If we do also add a new argument here, we should also add it to the docstring.